docs/source/tutorials/memory_optimizations.rst

```diff
@@ -8,14 +8,29 @@ Memory Optimization Overview
 
 torchtune comes with a host of plug-and-play memory optimization components which give you lots of flexibility
 to ``tune`` our recipes to your hardware. This page provides a brief glossary of these components and how you might use them.
+To make things easy, we've summarized these components in the following table:
+
+.. csv-table:: Memory optimization components
+   :header: "Component", "When to use?"
+   :widths: auto
+
+   ":ref:`glossary_precision`", "You'll usually want to leave this as its default bfloat16. If you're struggling with training stability or accuracy due to precision, fp32 may help."
+   ":ref:`glossary_act_ckpt`", "Use when you're memory constrained and need to handle larger batch sizes. Be aware that it may slow down training speed."
```

Comment:
Probably worth mentioning that this is specially useful for longer context lengths